entities.py

```diff
@@ -29,11 +63,26 @@ def __repr__(self):
         return f"{self.name} <{self.type}>@{self.pos}"
 
     def distance_to(self, dest):
-        """Return the distance this entity and the destination position."""
+        """
+        Calculate the distance between this entity and the destination position.
+
+        Args:
+            dest (Vector): Vector to calculate the distance to.
+
+        Returns:
+            Vector: Distance between this entity and the destination, in vector
+                form.
+        """
         return (self.pos - dest).norm
 
     def move_towards(self, dest, game_map):
-        """Move towards the destination if there's a clear path."""
+        """
+        Move towards the destination if there's a clear path.
```

Comment:
That actually made me realize that we don't really check if there's a clear path. But then again, I wonder if there's any case where there shouldn't be a clear path between any two points in the map (does a non-path-connected dungeon make sense?)